chalicelib/checks/helpers/lifecycle_utils.py

```diff
@@ -83,6 +83,7 @@ def check_file_lifecycle_status(
 
     search_query_base = (
         "/search/?type=File"
+        "&project.lifecycle_management_active=true"
```

Comment:
Maybe leave files to be uploaded by users out of this search as well to prevent deleting them if not uploaded within 2 weeks?
```suggestion
        "&project.lifecycle_management_active=true"
        "&status%21=uploading"
```

Reply:
Thanks, that's very helpful. In this case we definitely have to be more careful. I excluded `uploading` and `to be uploaded by workflow` from the search. This will solve the issue I encountered when testing on wolf and since files with that status are most likely not on S3 anyway, they will not waste space and money. 

All other files that are not on S3 keep they current lifecycle status, but I am updating the `last_checked` property, so that they are not constantly tested.

Does that work for you?